alerting/src/main/kotlin/com/amazon/opendistroforelasticsearch/alerting/util/AlertingUtils.kt

```diff
@@ -15,12 +15,16 @@
 
 package com.amazon.opendistroforelasticsearch.alerting.util
 
+import com.amazon.opendistroforelasticsearch.alerting.destination.message.BaseMessage
 import com.amazon.opendistroforelasticsearch.alerting.model.destination.Destination
 import com.amazon.opendistroforelasticsearch.alerting.settings.DestinationSettings
 import com.amazon.opendistroforelasticsearch.commons.authuser.User
 import org.elasticsearch.ElasticsearchStatusException
 import org.elasticsearch.action.ActionListener
 import org.elasticsearch.rest.RestStatus
+import inet.ipaddr.IPAddressString
+import java.net.InetAddress
+import org.elasticsearch.transport.TransportChannel.logger
```

Comment:
I think incorrect logger used below?

Reply:
Addressed in https://github.com/opendistro-for-elasticsearch/alerting/pull/360